### PR TITLE
common: misc cleanups in common, global, os, osd submodules

### PR DESCRIPTION
**Pull request by Yan-waller**
Signed-off-by: Yan Jun yan.jun8@zte.com.cn

### CODE REVIEW

File: src/global/global_init.cc [hunk 1]
```diff
@@ -114,7 +114,7 @@ void global_pre_init(std::vector < const char * > *alt_def_args,
         dout_emergency(oss.str());
         _exit(1);
       } else {
-        derr <<"did not load config file, using default settings." << dendl;
+        derr << "did not load config file, using default settings." << dendl;
```
Comment (joscollin) [inline src/global/global_init.cc]: Revert this unwanted character.

File: src/os/filestore/FileStore.cc [hunk 2]
```diff
@@ -907,7 +907,7 @@ int FileStore::mkfs()
 #if defined(__linux__)
   if (basefs.f_type == BTRFS_SUPER_MAGIC &&
       !g_ceph_context->check_experimental_feature_enabled("btrfs")) {
-    derr <<__FUNC__ << ": deprecated btrfs support is not enabled" << dendl;
+    derr << __FUNC__ << ": deprecated btrfs support is not enabled" << dendl;
```
Comment (joscollin) [inline src/os/filestore/FileStore.cc]: Revert this unwanted character.

File: src/global/global_init.cc (hunk 1 above)
Comment (Yan-waller) [inline src/global/global_init.cc]: sorry, but why ? I'm confused.

File: src/os/filestore/FileStore.cc (hunk 2 above)
Comment (Yan-waller) [inline src/os/filestore/FileStore.cc]: why

**Comment by joscollin** [comment]
@Yan-waller Oh, I think I read it wrongly. My bad.

**Comment by Yan-waller** [comment]
@joscollin okay, done.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-13_20:38:15-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**